README.md

```diff
@@ -90,7 +91,7 @@ For example if the play services are not installed it will prompt:
 
 #### - `configure(configuration)`
 
-It is mandatory to call this method before `signIn()` and `signInSilently()`. This method is sync meaning you can call `singIn` right after it. In typical situations this needs to be called only once.
+It is mandatory to call this method before attempting to call `signIn()` and `signInSilently()`. This method is sync meaning you can call `singIn` right after it. In typical scenarios this needs to be called only once, after your app starts.
```

Comment:
Typo: `singIn` should be `signIn`. I think this is mine. :P

Reply:
fixed